File: app/src/main/java/com/example/homepharmacy/MainActivity.java
package com.example.homepharmacy;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.navigation.NavController;
import androidx.navigation.NavDirections;
import androidx.navigation.Navigation;
import androidx.navigation.fragment.NavHostFragment;
import androidx.navigation.ui.AppBarConfiguration;
import androidx.navigation.ui.NavigationUI;

import android.content.ClipData;
import android.os.Bundle;
import android.view.MenuItem;

import com.google.android.material.bottomnavigation.BottomNavigationView;

import java.util.Objects;

public class MainActivity extends AppCompatActivity {
    private static AppBarConfiguration mainActivityToolbarConfiguration;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Toolbar mainActivityToolbar = findViewById(R.id.main_activity_toolbar);
        setSupportActionBar(mainActivityToolbar);

        BottomNavigationView mainBottomNavigator = findViewById(R.id.main_activity_bottom_navigation);
        mainBottomNavigator.setItemIconTintList(null);

        NavHostFragment navHostFragment = (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.nav_main_activity_host_fragment);
        NavController navController = Objects.requireNonNull(navHostFragment).getNavController();
        NavigationUI.setupWithNavController(mainBottomNavigator, navController);
        mainActivityToolbarConfiguration = new AppBarConfiguration.Builder(R.id.drugs_destination_fragment, R.id.calendar_destination_fragment).build();
        NavigationUI.setupActionBarWithNavController(this, navController, mainActivityToolbarConfiguration);

    }
}

File: app/src/main/java/MainActivity.kt
package com.example.homepharmacy

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.fragment.NavHostFragment
import androidx.navigation.ui.NavigationUI
import com.google.android.material.bottomnavigation.BottomNavigationView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val navHostFragment = supportFragmentManager.findFragmentById(R.id.nav_main_activity_host_fragment) as NavHostFragment?
        val navController = navHostFragment!!.navController //getting navController for navigation in the app

        val mainBottomNavigator = findViewById<BottomNavigationView>(R.id.main_activity_bottom_navigation)
        mainBottomNavigator.itemIconTintList = null //setting icon color to null, because otherwise they would whole
        NavigationUI.setupWithNavController(mainBottomNavigator, navController) //connecting bottomNavigationView to navController for it's auto update
    }
}